### PR TITLE
Add hardware specifications section to pax readme

### PR DESCRIPTION
**Pull request by ashors1**

### CODE REVIEW

File: rosetta/rosetta/projects/pax/README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 Any `paxml/*` or `praxis/*` relative directory/file can be found in [google/paxml](https://github.com/google/paxml/tree/main) or [google/praxis](https://github.com/google/praxis/tree/main), respectively, but to
 view the most up-to-date version of that directory/file with any GPU-specific patches, please see [Inspecting the source code](#inspecting-the-source-code).
 
+## Hardware Specifications
+Convergence and performance has been validated on NVIDIA DGX A100 (8x A100 80G) nodes. We provide both singlenode and multinode pre-training support. If running on a machine with less than 80G memory, some of the default configurations may run out of memory; if you run out of memory and have more GPUs available, increase your GPU count and decrease your batch size per GPU.
```
Comment (yhtang) [inline rosetta/rosetta/projects/pax/README.md:8]: It would be great if we can link to some artifacts from the claimed successful run.

Comment (ashors1) [reply rosetta/rosetta/projects/pax/README.md:8]: Convergence logs are linked in the [configs section](https://github.com/NVIDIA/JAX-Toolbox/blob/98278cc443a03f4089a505f83b6f29ec43097fc2/rosetta/rosetta/projects/pax/README.md#configs). Is this sufficient, or do you think we should upload additional aritfacts?

Comment (yhtang) [reply rosetta/rosetta/projects/pax/README.md:8]: How about just say something like
> Convergence and performance has been validated on NVIDIA DGX A100 (8x A100 80G) nodes. For details, see the [Configs](#configs) section below.
?

Comment (yhtang) [reply rosetta/rosetta/projects/pax/README.md:8]: Here, we use Markdown anchors to link to the Config section on the same page.

Comment (ashors1) [reply rosetta/rosetta/projects/pax/README.md]: Sounds good, I made that change

**Review by yhtang — APPROVED** [review]
LGTM

**Pull request merged**